rules/framework.bzl

```diff
@@ -522,12 +522,25 @@ def _copy_swiftmodule(ctx, framework_files):
     # only add a swift module to the SwiftInfo if we've actually got a swiftmodule
     swiftmodule_name = paths.split_extension(inputs.swiftmodule.basename)[0]
 
+    # find the first swift symbol_grap in the direct_modules of the deps of this target
+    # and add it to the swift info
+    symbol_graph = None
+    for dep in ctx.attr.deps[1:]:
+        if SwiftInfo in dep:
+            for module in dep[SwiftInfo].direct_modules:
+                if module.swift and module.swift.symbol_graph:
+                    symbol_graph = module.swift.symbol_graph
+                    break
+            if symbol_graph:
+                break
```

Comment:
Not sure this is totally correct (because it looks different than how we do it for the other fields), but it works for the examples I tried.